### PR TITLE
Add license for Beantown Bash

### PR DESCRIPTION
**Pull request by kvnyng**

### CODE REVIEW

**Comment by kvnyng** [comment]
@sampoder @exu3 Sorry for taking so long! Here you go!

**Comment by sampoder** [comment]
All good, LGTM!

**Review by sampoder — APPROVED** [review]

**Comment by sampoder** [comment]
Rip GitHub mobile won't let me resolve conflicts, text me and I'll do this when I get home!

**Comment by exu3** [comment]
Seems like I can't resolve conflicts either since I don't have write access on this forked branch.

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Every year, around the world, Hack Clubbers organise [high-school hackathons](ht
 | CodeDay Singapore | [photos.app.goo.gl/pd...](https://photos.app.goo.gl/pd5MprLVn7ZDixv17) | Mar. 2022 | Singapore (SG) | [CC0 (“No Rights Reserved”)](https://creativecommons.org/publicdomain/zero/1.0/) | [@sampoder](https://github.com/sampoder) |
 | Flagship | [photos.app.goo.gl/F5...](https://photos.app.goo.gl/F5JDxq5FzfhofTMD9) | Jun. 2019 | San Francisco (USA) | [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/) | [@zachlatta](https://github.com/zachlatta) |
 | Forsyth Hacks | [photos.app.goo.gl/pi...](https://photos.app.goo.gl/piCyiv9YJxfyKUWj8) | Mar. 2023 | Georgia (USA) | [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/) | [@sahitid](https://github.com/sahitid) |
```
Comment (exu3) [inline README.md:14]:
```suggestion
| Forsyth Hacks 2.0 | [photos.app.goo.gl/Rd...](https://photos.app.goo.gl/RdvvgMmpD3rpioNc6) | Nov. 2023 | Georgia (USA) | [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/) | [@sahitid](https://github.com/sahitid) |
```

**Pull request closed without merging**